src/app/fdctl/run/run_solana.c

```diff
@@ -85,6 +85,7 @@ solana_labs_main( void * args ) {
 
   /* ledger */
   ADD( "--ledger", config->ledger.path );
+  ADD( "--accounts", config->ledger.accounts_path );
```

Comment:
Thanks for the fix! I think we should only pass this if `if( !strcmp( "", config->ledger.accounts_path ) )`

Reply:
done!